lib/ravelin/client.rb

```diff
@@ -0,0 +1,66 @@
+module Ravelin
+  class Client
+    API_BASE = 'https://api.ravelin.com'
+
+    def initialize(api_key:)
+      @api_key = api_key
+
+      @connection = Faraday.new(API_BASE, faraday_options) do |conn|
+        conn.response :json, context_type: /\bjson$/
+        conn.adapter Ravelin.faraday_adapter
+      end
+    end
+
+    def send_event(**args)
+      event = Event.new(**args)
+
+      post("/v2/#{event.name}", event.serializable_hash)
+    end
+
+    def send_backfill_event(**args)
+      unless args.has_key?(:timestamp)
```

Comment:
I think I would do this if the `timestamp` argument was used in this method. Since the arguments are passed on to the `Event`, I'd prefer to be explicit here.